GSoC-2025/Ethology: Low-Shot Ethology Detection (Adithya Sau)

```diff
@@ -0,0 +1,95 @@
+# Ethology: Low-Shot Ethology Detection (Adithya Sau)
+
+## **Personal Details**
+
+- **Full Name:** Adithya Sau  
+- **Email:** adithyasau@gmail.com  
+- **GitHub Username:** https://github.com/Adisauz  
+- **Zulip Username:** https://neuroinformatics.zulipchat.com/#recent_topics  
+- **Location & Time Zone:** Pune, India (UTC+5:30)  
+
+---
+
+## **Code Contribution**
+
+https://github.com/brainglobe/brainglobe.github.io/pull/314
+-Added an Examples of brainreder in sphinx documentation
+https://github.com/neuroinformatics-unit/ethology/pull/82
+-Added a python program that can extract evenly spaced frames over a specified interval.
+https://github.com/neuroinformatics-unit/ethology/pull/81
+-Added a python program and can track object(cars) when user selects on the detected object
+
+---
+- **Proposal discussion link:** [Link to discussion PR]
```

Comment:
Remember to add the link to this PR here as per [the template](https://github.com/neuroinformatics-unit/gsoc/blob/main/GSoC-2025/application_template.md)